docs/api/customers.rst

```diff
@@ -139,9 +139,7 @@ Exemplary Response
         "firstName":"John",
         "lastName":"Diggle",
         "gender":"m",
-        "group":[
-
-        ]
+        "group":{}
```

Comment:
Shouldn't it be `"group":null` or `"group":"GROUP_CODE"` then?

Reply:
That's the response given by the API right now

Reply:
Perhaps we are missing serialization definition of customer groups.

Reply:
That's the current behaviour that needs to be documented anyway :)